mission-control/docs/registry/kubernetes.md

```diff
@@ -10,6 +10,31 @@ helm repo update
 helm install {release-name} flanksource/mission-control-kubernetes
 ```
 
+After running `helm install` you should get a success message:
+
+```sh
+NAME: {release-name}
+LAST DEPLOYED: Thu Feb 14 19:00:32 2024
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Kubernetes topology and scraper added
+```
+
+When you go to the dashboard now, you can now see a cluster topology:
```

Comment:
Can you check the queries and/or use AWS as those metrics don't look right. Also see https://github.com/flanksource/docs/blob/main/CONTRIBUTING.md#screenshots